### PR TITLE
Add storyAct slice of state.session to track story progress

### PR DESCRIPTION
**Pull request by ningyuansg**
Also fixed some broken links.

### CODE REVIEW

File: src/components/academy/game/create-initializer.js [hunk 1]
```diff
@@ -7,14 +7,14 @@ export default function (StoryXMLPlayer, story, username, attemptedAll) {
 
   var hookHandlers = {
     startMission: function (number) {
-      document.getElementById('attempt').click()
+      history.push(`/academy/#{number}`)
```
Comment (remo5000) [inline src/components/academy/game/create-initializer.js]: Is there a way to get the assessment type here? It needs it for the routing. If not, we can add a separate switch for this or remove the type in the routing entirely.

File: src/components/academy/game/index.tsx [hunk 2]
```diff
@@ -2,16 +2,18 @@ import * as React from 'react'
...
+export type DispatchProps = {
   handleSaveCanvas: (c: HTMLCanvasElement) => void
 }
 
-export type DispatchProps = Pick<IGameProps, 'handleSaveCanvas'>
-
-export type StateProps = Pick<IGameProps, 'canvas'>
+export type StateProps = {
+  canvas?: HTMLCanvasElement
+  storyAct: string
```
Comment (remo5000) [inline src/components/academy/game/index.tsx]: Is there a way to check the validity of the storyAct within the game?

File: src/reducers/states.ts [hunk 3]
```diff
@@ -181,14 +183,14 @@ export const defaultSession: ISessionState = {
...
-  assessmentOverviews: undefined,
   assessments: new Map<number, IAssessment>(),
+  assessmentOverviews: undefined,
   historyHelper: {
     lastAcademyLocations: [null, null],
     lastGeneralLocations: [null, null]
   },
-  accessToken: undefined,
   refreshToken: undefined,
+  storyAct: 'mission-1',
```
Comment (remo5000) [inline src/reducers/states.ts]: Can the fetching of a user's current story status be made an issue?

File: src/components/academy/game/create-initializer.js (hunk 1 above)
Comment (ningyuansg) [reply src/components/academy/game/create-initializer.js]: Fixed with 8c262e7.

**Pull request merged**